src/style.css

```
@@ -1 +1,132 @@
-/* empty source */
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+}
+
+body {
+  margin: 0;
+  min-height: 100vh;
+
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.card__description {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-accent);
+  margin-bottom: 4px;
+}
+
+/* #region Product Code */
+.card__product-code {
+  color: var(--secondary);
+  margin-bottom: 16px;
+}
+/* #endregion */
+
+/* #region Review */
+.card__review-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__review-text {
+  color: var(--main-accent);
+}
+
+/* #endregion */
+
+/* #region Price */
+.card__price-wrapper {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-label {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-accent);
+}
+/* #endregion */
+
+/* #region Button */
+.card__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+  background-color: var(--blue-accent);
+  border: 1px solid var(--blue-accent);
+  box-sizing: border-box;
+  border-radius: 5px;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.card__link:hover {
+  background-color: #fff;
+  border: 1px solid var(--blue-accent);
+  color: var(--blue-accent);
+}

```

No need to set border  again with the same values here(when you hover on the card) because it is already set in card__link.
```suggestion
.card__link:hover {
  background-color: #fff;
  color: var(--blue-accent);
}

```

---

Please change those things that i left in comments.
The rest of your code is clean !